pyproject.toml

```diff
@@ -36,22 +36,25 @@ dependencies = [
     "dill",
     "hydra-core",
     "numpy",
-    "pybind11",
-    "ray>=2.38",
-    "tensordict",
-    "transformers<4.48",
-    "vllm<=0.6.3",
+    "pandas",
     "peft",
-    "liger-kernel",
+    "pyarrow>=15.0.0",
+    "pybind11",
     "pylatexenc",
+    "ray>=2.10",
```

Comment:
This should be ray>= 2.38

Reply:
since setup.py and toml specifies the minimal requirements, i think 2.10 is also fine? qwen team has some issues using ray 2.38 so i want to be less restrictive here

Reply:
It looks fine to me